perception/autoware_detected_object_validation/src/lanelet_filter/lanelet_filter.cpp

```diff
@@ -146,6 +152,17 @@ void ObjectLaneletFilterNode::objectCallback(
     RCLCPP_ERROR(get_logger(), "Failed transform to %s.", lanelet_frame_id_.c_str());
     return;
   }
+  // vehicle base pose :map -> base_link
+  try {
+    ego_base_height_ = tf_buffer_
+                         .lookupTransform(
+                           lanelet_frame_id_, "base_link", transformed_objects.header.stamp,
+                           rclcpp::Duration::from_seconds(0.5))
+                         .transform.translation.z;
+  } catch (const tf2::TransformException & ex) {
+    RCLCPP_ERROR_STREAM(get_logger(), "Failed to get transform: " << ex.what());
+    return;
+  }
```

Comment:
IMO, it is better to add condition of `use_height_threshold` here also
```suggestion
    // vehicle base pose :map -> base_link
  if (filter_settings_.use_height_threshold) {
    try {
      ego_base_height_ = tf_buffer_
                          .lookupTransform(
                            lanelet_frame_id_, "base_link", transformed_objects.header.stamp,
                            rclcpp::Duration::from_seconds(0.5))
                          .transform.translation.z;
    } catch (const tf2::TransformException & ex) {
      RCLCPP_ERROR_STREAM(get_logger(), "Failed to get transform: " << ex.what());
      return;
    }
  }
```

Reply:
Thanks! Fixed in https://github.com/autowarefoundation/autoware.universe/pull/10003/commits/7206f1d80c8c6cb145ceeeff76c46b0d691441fb.